src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/service/RecommendationService.kt

```diff
@@ -63,17 +64,28 @@ internal class RecommendationService(
 
   suspend fun createRecommendation(
     recommendationRequest: CreateRecommendationRequest,
-    username: String?
+    username: String?,
+    featureFlags: FeatureFlags?
   ): RecommendationResponse {
     val userAccessResponse = recommendationRequest.crn?.let { userAccessValidator.checkUserAccess(it) }
     if (userAccessValidator.isUserExcludedRestrictedOrNotFound(userAccessResponse)) {
       throw UserAccessException(Gson().toJson(userAccessResponse))
     } else {
       val personDetails = recommendationRequest.crn?.let { personDetailsService.getPersonDetails(it) }
+      val status = if (featureFlags?.flagConsiderRecall == true) Status.RECALL_CONSIDERED else Status.DRAFT
```

Comment:
Don't we need a change on the update endpoint to change the status from `RECALL_CONSIDERED` to `DRAFT` when they click on 'Make recommendation' button after concerns have been raised? Or is that covered by separate ticket?

Reply:
I suppose it will be part of the update work.